crypto/crypto_test.go

```
@@ -4,7 +4,7 @@ import "testing"
 
 // copied from official crypto.ed25519 tests
 func TestSignVerify(t *testing.T) {
-	key := GenerateKey()
+	key, _ := GenerateKey()
```

Let's not ignore the error.